src/components/example/ExampleSearchInput.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="exampleSearchInput">
```

Comment:
Follow css naming class pattern used in project.

Replace `exampleSearchInput` for `example-searchs-input`